cubeseed_login/.husky/pre-commit

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+cd cubeseed_login
+# npx pretty-quick --staged
+# npm run pre-commit
```

Comment:
Shouldn't you uncomment those? 
What does the husky.sh do?
Where is it?

Reply:
I will, [Husky](https://typicode.github.io/husky/) prevent commits till all the linting issues are fixed.

Reply:
But where is the husky.sh file? and what's in it?

Reply:
It is created when you run npm run prepare

Reply:
closes #24